a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -359,3 +360,196 @@ def revert(self, vthunder, *args, **kwargs):
         except Exception as e:
             LOG.exception("Failed to revert partition create : %s", str(e))
             raise
+
+
+class TagEthernetBaseTask(task.Task):
+
+    def __init__(self, **kwargs):
+        super(TagEthernetBaseTask, self).__init__(**kwargs)
+        self._network_driver = None
+        self._subnet = None
+
+    @property
+    def network_driver(self):
+        if self._network_driver is None:
+            self._network_driver = a10_task_utils.get_a10_network_driver()
+        return self._network_driver
+
+    def get_tag_interface_info(self, project_id, vlan_id):
+        if project_id in CONF.rack_vthunder.devices:
+            vthunder_conf = CONF.rack_vthunder.devices[project_id]
+            if vthunder_conf.interface_vlan_map is not None:
```

Comment:
You can keep this as  `if vthunder_conf.interface_vlan_map:` also

Reply:
Done.